pkg/operator/ceph/controller/predicate.go

```diff
@@ -528,6 +516,32 @@ func WatchPredicateForNonCRDObject(owner runtime.Object, scheme *runtime.Scheme)
 					return false
 				}
 
+				// If the resource is an OBC we check only for changes needed for bucket notifications
+				// meaning: label changes, actual bucket name was set or state changes to "bound"
+				newObj, ok := e.ObjectNew.(*bktv1alpha1.ObjectBucketClaim)
+				if ok {
+					oldObj, _ := e.ObjectOld.(*bktv1alpha1.ObjectBucketClaim)
+					diff := cmp.Diff(
```

Comment:
`reflect.DeepEqual` can be used to compare the string maps

Reply:
but i want to compare only changes in labels, not any change in the new/old object

Reply:
I mean reflect.DeepEqual on the `label` string map. Not on the entire object

Reply:
ok. will try that